lib/firefox/webdriver/index.js

```diff
@@ -62,7 +62,9 @@ module.exports.configureBuilder = function(builder, baseDir, options) {
   // Output the window recorder image frames to a base directory.
   // This pref expects a trailing slash.
   if (firefoxConfig.windowRecorder) {
-    profile.setPreference('layers.windowrecording.path', path.join(baseDir,'/'));
+    // FIXME this shouldn't hard code path to external storage
+    const dir = !firefoxConfig.android ? baseDir : '/sdcard/browsertime-firefox-windowrecording';
```

Comment:
I think `cond ? trueCase : falseCase` is easier to grok than `!cond ? falseCase : trueCase`

Reply:
I generally prefer "standard flow" ? "standard case" : "exceptional case", but I'll change this.